docs/STEPS.md

```diff
@@ -886,25 +886,25 @@ Support
 |:-----:|:--------------:|:----------:|:--------------:|:----------:|
 |   ✔   |       ✔        |     ✔      |       ?        |     ?      |
 
```

Comment:
Could you also add the
`## Move over \<by> = \<by_value>`
doc, and just link it and explain that it does the same? On second thought I think we should not deprecate the `hover over`, because it is a common expression and it might make this step easier to find.